src/burp/restore.epp

```diff
@@ -2721,6 +2725,24 @@ bool get_chk_constraint(BurpGlobals* tdgbl)
 	return true;
 }
 
+static void clearAttributes(BlobBuffer& attr)
+{
+	charset cs;
+	IntlUtil::initAsciiCharset(&cs);
+	AutoPtr<CharSet> ascii(Firebird::CharSet::createInstance(*getDefaultMemoryPool(), 0, &cs));
+
+	IntlUtil::SpecificAttributesMap map;
+	IntlUtil::parseSpecificAttributes(ascii, attr.getCount(), attr.begin(), &map);
+
+	map.remove("ICU-VERSION");
+	map.remove("COLL-VERSION");
+	string currentIcu = UnicodeUtil::getDefaultIcuVersion();
+	map.put("ICU-VERSION", currentIcu);
```

Comment:
Isn't this going to add ICU tags even to non ICU collations?

Reply:
Hops - will fix.